.github/workflows/test.yml

```diff
@@ -65,4 +65,50 @@ jobs:
         python -m pytest -v --durations=0 -x test_trainable.py
         declare -a arr=("AsyncHyperBandScheduler" "HyperBandScheduler" "MedianStoppingRule" "ASHAScheduler"); for s in "${arr[@]}"; do python schedulers.py --scheduler "$s"; done
         cd ../examples
-        for f in *.py; do echo "running $f" && python "$f" || exit 1 ; done
\ No newline at end of file
+        for f in *.py; do echo "running $f" && python "$f" || exit 1 ; done
+
+  build_docs:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        git clone https://github.com/ray-project/ray
```

Comment:
You can use `actions/checkout@v2` for it as well:
```
- uses: actions/checkout@v2
  with:
    repository: 'ray-project/ray'
	path: 'ray'
```
But tbh it's not a big deal, considering that it's a public repo. Maybe consider doing `--depth 1` to just fetch the latest commit instead of the entire history - it'll speed things up as we don't need anything but the latest commit to do an install (`action/checkout` does that by default).

Reply:
Hmm, yeah this sounds like a good idea.